src/createAnimatedComponent.tsx

```diff
@@ -479,6 +595,17 @@ export default function createAnimatedComponent(
                 };
               }
               return this.initialStyle;
+            } else if (hasInlineStyles(style)) {
+              const newStyle: StyleProps = {};
+              for (const [key, styleValue] of Object.entries(style)) {
+                if (
+                  styleValue.value === undefined &&
```

Comment:
can we define a method `isSharedValue` and use it here? I was thinking about adding such method for some time and it'll be useful in other places as well.